bot/seasons/evergreen/issues.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Issues(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=("issues",))
+    async def issue(self, ctx, number: int, repository: str = "seasonalbot", user: str = "python-discord"):
+        """Command to retrieve issues from a GitHub repository."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
+        failed_status = {
+            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
+            403: f"Rate limit exceeded. Please wait a while before trying again!"
+        }
+
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
+            response_code = r.status
+
+        if response_code in failed_status:
+            return await ctx.send(failed_status[response_code])
+
+        issue_embed = discord.Embed(colour=0x00ff37)
+        issue_embed.add_field(name="Repository", value=f"{user}/{repository}", inline=False)
```

Comment:
The value here could link to the repository directly using the markdown hyperlink syntax
```suggestion
        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}](json_data['repository_url'])", inline=False)
```
This is distinct from the `Link` field as that links to the specific issue, while this one would merely link to the repository.